platform/cavium/docker-syncd-cavm/Dockerfile.template

```diff
@@ -0,0 +1,18 @@
+FROM docker-base
+
+RUN apt-get update
+
+COPY deps /deps
+
+RUN apt-get -y install libpcap-dev libxml2-dev python-dev swig libsensors4-dev redis-tools
```

Comment:
redis-tools is installed from sources, not needed here
Why are dev packages needed?